### ISSUE TITLE
How is the.pkl file generated in print_font_template in Stroke-level-decomposition?

### ISSUE THREAD

**Issue by cptbtptp125** [FudanVI/FudanOCR#64]
How is the.pkl file generated in print_font_template in Stroke-level-decomposition? Is there any corresponding support sample for stroke level recognition in Korean?I would appreciate it if you could help me.

**Comment by Hongtauo** [comment]
同问，代码中的数据加载器部分，加载的文件.pkl或者lmdb文件的格式是什么？能否给个示例？感谢作者！
`def get_dataloader(root, shuffle=False):
    if root.endswith('pkl'):
        f = open(root, 'rb')
        dataset = pkl.load(f)
        dataloader = torch.utils.data.DataLoader(
            dataset, batch_size=config['batch'], shuffle=shuffle, num_workers=8,
        )
    else:
        dataset = lmdbDataset(root, resizeNormalize((config['image_size'], config['image_size'])),
                              alphabet=alphabet_character)
        dataloader = torch.utils.data.DataLoader(
            dataset, batch_size=config['batch'], shuffle=shuffle, num_workers=8,
        )
    return dataloader, dataset`
